gms/api/src/main/pegasus/com/linkedin/glossary/GlossaryTermKey.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.glossary
+
+import com.linkedin.common.GlossaryNodeUrn
+
+/**
+ * Key for GlossaryTerm resource
+ */
+record GlossaryTermKey {
+
+  @validate.strlen = {
+    "max" : 20,
+    "min" : 1
+  }
+  name: string
+
+  /**
+   * Parent business nodes(in sequence) to this business term.
+   */
+  glossaryNodePath: string
```

Comment:
this should be `parentNode: GlossaryNodeUrn`

Reply:
on second thoughts, okay to be the string path... still a bit confused about why this is different from the main GlossaryTermUrn

Reply:
We forgot to remove this field after deciding on the glossary term urn format. GlossaryNodeUrn has 1 field name, GlossaryTermKey will also have 1 field name. Will remove `glossaryNodePath `

Reply:
done